testing/tests/DevExpress.ui.widgets.dataGrid/dataGrid.export.tests.js

```diff
@@ -0,0 +1,882 @@
+                        { dataField: "field3", dataType: "string" },
+                    ]
+                }
+            ],
+            dataSource: [{ field1: 'str1', field2: 'str2', field3: 'str3' } ],
+        },
+        { styles, worksheet, sharedStrings }
+    );
+});
+
+QUnit.test("Summary - total", function(assert) {
+    const styles = STYLESHEET_HEADER_XML +
+        '<numFmts count="0"></numFmts>' +
+        internals.BASE_STYLE_XML +
+        '<cellXfs count="5">' +
+        '<xf xfId="0" applyAlignment="1" fontId="1" applyNumberFormat="0" numFmtId="0"><alignment vertical="top" wrapText="1" horizontal="center" /></xf>' +
+        '<xf xfId="0" applyAlignment="1" fontId="1" applyNumberFormat="0" numFmtId="0"><alignment vertical="top" wrapText="1" horizontal="left" /></xf>' +
+        '<xf xfId="0" applyAlignment="1" fontId="1" applyNumberFormat="0" numFmtId="0"><alignment vertical="top" wrapText="1" horizontal="right" /></xf>' +
+        '<xf xfId="0" applyAlignment="1" fontId="0" applyNumberFormat="0" numFmtId="0"><alignment vertical="top" wrapText="0" horizontal="right" /></xf>' +
+        '<xf xfId="0" applyAlignment="1" fontId="1" applyNumberFormat="0" numFmtId="0"><alignment vertical="top" wrapText="0" horizontal="left" /></xf>' +
+        '</cellXfs>' +
+        STYLESHEET_FOOTER_XML;
+    const worksheet = internals.WORKSHEET_HEADER_XML +
+        '<sheetPr/><dimension ref="A1:C1"/>' +
+        '<sheetViews><sheetView tabSelected="1" workbookViewId="0"><pane activePane="bottomLeft" state="frozen" ySplit="1" topLeftCell="A2" /></sheetView></sheetViews>' +
+        '<sheetFormatPr defaultRowHeight="15" outlineLevelRow="0" x14ac:dyDescent="0.25"/>' +
+        '<cols><col width="13.57" min="1" max="1" /></cols>' +
+        '<sheetData>' +
+        '<row r="1" spans="1:1" outlineLevel="0" x14ac:dyDescent="0.25"><c r="A1" s="0" t="s"><v>0</v></c></row>' +
+        '<row r="2" spans="1:1" outlineLevel="0" x14ac:dyDescent="0.25"><c r="A2" s="3" t="n"><v>1</v></c></row>' +
+        '<row r="3" spans="1:1" outlineLevel="0" x14ac:dyDescent="0.25"><c r="A3" s="3" t="n"><v>2</v></c></row>' +
+        '<row r="4" spans="1:1" outlineLevel="0" x14ac:dyDescent="0.25"><c r="A4" s="2" t="s"><v>1</v></c></row>' +
+        '</sheetData>' +
+        '<ignoredErrors><ignoredError sqref="A1:C4" numberStoredAsText="1" /></ignoredErrors></worksheet>';
+    const sharedStrings = SHARED_STRINGS_HEADER_XML + ' count="2" uniqueCount="2">' +
+        '<si><t>Field 1</t></si>' +
+        '<si><t>Count: 2</t></si>' +
+        '</sst>';
+
+    testConfiguration(
+        assert,
+        {
+            columns: [{ dataField: "field1", dataType: "number" }],
+            dataSource: [{ field1: 1 }, { field1: 2 }],
+            summary: {
+                totalItems: [{ column: 'field1', summaryType: 'count' }]
+            }
+        },
+        { styles, worksheet, sharedStrings }
+    );
+});
+
+QUnit.test("Summary - groupping", function(assert) {
```

Comment:
Replace this to "Group total summary"

Reply:
Total summaries export is tested in the "Total summary" test.
Group summaries export is tested in the "Group summary", "Group summary - alignByColumn: true", "Group summary - showInGroupFooter: true" tests.